apps/server/lib/bootstrap.ts

```diff
@@ -156,28 +160,27 @@ export const bootstrap = async (context, options) => {
 	await webServer.start();
 
 	logger.info(context, 'Setting up cache');
-	const cache = new core.MemoryCache(environment.redis);
+	const cache = new Cache(environment.redis);
 	if (cache) {
 		await cache.connect();
 	}
 
 	// Instantiate a core instance that will handle DB operations
-	const backendOptions =
+	const backendOptions: PostgresBackendOptions =
 		options && options.database
 			? Object.assign({}, environment.database.options, options.database)
 			: environment.database.options;
-	const jellyfish = (await core.create(context, cache, {
-		backend: backendOptions,
-	})) as any as coreType.JellyfishKernel;
+
+	const jellyfish = await Kernel.withPostgres(context, cache, backendOptions);
 
 	const metricsServer = metrics.startServer(
 		context,
 		environment.metrics.ports.app,
 	);
 
 	// Create queue instances
-	const producer = new Producer(jellyfish, jellyfish.sessions.admin);
-	const consumer = new Consumer(jellyfish, jellyfish.sessions.admin);
+	const producer = new Producer(jellyfish as any, jellyfish.sessions!.admin);
```

Comment:
This takes a `Kernel` object. Why is typechecking failing?

Reply:
@Ereski it's failing because the queue module has not yet been updated, and is still expecting a `JellyfishKernel` type.